README.md

```diff
@@ -37,6 +37,12 @@ For more information on rule processing, see the [AWS User Guide](https://docs.a
 ![Component Diagram](docs/component.diagram.png)
 
 ### Template Parameters
+The following template parameters are used to configure Cloudformation resources
+| Template Parameter | Description | Example Value |
+| --- | --- | --- |
+| DnsName | Custom host name to use for API Gateway | cost-rules.example.com |
+| AcmCertificateArn | AWS ARN of an ACM certificate valid for `DnsName` | arn:aws:acm:us-east-1:&lt;ACCOUNT ID>:certificate/&lt;CERTIFICATE UUID> |
```

Comment:
shouldn't these be added to the `install` section as well?

Reply:
whoops, yeah